apps/native-component-list/webpack.config.js

```diff
@@ -6,25 +6,27 @@ const ModuleScopePlugin = require('react-dev-utils/ModuleScopePlugin');
 const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const ManifestPlugin = require('webpack-manifest-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const WorkboxPlugin = require('workbox-webpack-plugin');
 const pckg = require('./package.json');
 
 const absolutePath = location => path.resolve(__dirname, location);
 
 const locations = {
   // Shouldn't change
   root: absolutePath('.'),
-  output: absolutePath('web'),
+  output: absolutePath('build'),
```

Comment:
“build” is too common of a name since people might emit non-web-specific code to a “build” directory like we do for TS unimodules — picking a more web-specific name would work better here I think, even “web-build”